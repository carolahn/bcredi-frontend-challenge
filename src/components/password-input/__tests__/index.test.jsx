import React from "react";
import renderer from "react-test-renderer";

import PasswordInput from "../index";

describe("PasswordInput render test", () => {
  it("renders correctly PasswordInput", () => {
    const tree = renderer
      .create(<PasswordInput values={{}} register={() => {}} errors={{}} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
