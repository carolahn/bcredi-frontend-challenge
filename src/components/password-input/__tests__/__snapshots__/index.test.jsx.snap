// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PasswordInput render test renders correctly PasswordInput 1`] = `
<div
  className="register__input-wrapper"
>
  <label
    className="register__label"
  >
    Senha
  </label>
  <div
    className="register__input--toggle-eye"
  >
    <input
      className="sc-gsTCUz fKZdNo register__input"
      name="password"
      placeholder="Cadastre uma senha"
      style={
        Object {
          "border": "1px solid #e6e6e6",
          "color": "#333333",
        }
      }
      type="password"
      width="320px"
    />
    <i
      className="register__icon--eye"
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-eye fa-w-18 "
        data-icon="eye"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 576 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M572.52 241.4C518.29 135.59 410.93 64 288 64S57.68 135.64 3.48 241.41a32.35 32.35 0 0 0 0 29.19C57.71 376.41 165.07 448 288 448s230.32-71.64 284.52-177.41a32.35 32.35 0 0 0 0-29.19zM288 400a144 144 0 1 1 144-144 143.93 143.93 0 0 1-144 144zm0-240a95.31 95.31 0 0 0-25.31 3.79 47.85 47.85 0 0 1-66.9 66.9A95.78 95.78 0 1 0 288 160z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
    </i>
  </div>
</div>
`;
